packages/core/package.json

```
@@ -39,7 +39,7 @@
     "copy-webpack-plugin": "^6.1.0",
     "core-js": "3.6.5",
     "css-loader": "^4.2.2",
-    "dotenv-webpack": "^3.0.0",
+    "dotenv-webpack": ">=2.0.0 <3.0.0",
```

```suggestion
    "dotenv-webpack": "^2.0.0",
```
Because Dependabot is merged manually, I don't think we need to go this far as we won't get notification for minors or patches. Also, if another PR is opened by Dependabot for dotenv-webpack, we can `@dependabot ignore` as a comment to manually set an ignore.

Make sense? Thoughts otherwise?

---

^^ @dthyresson

---

I think the issue was that Dependabot ran and then it got into a canary release -- so while they may be manually merged, it still got in to that.

If this fixes that via process, then ok.

My question was, however:

```
"dotenv-webpack": "^2.0.0",

which should keep the version between under 3.

https://classic.yarnpkg.com/en/docs/dependency-versions#toc-caret-ranges

-= 2.0.0 < 3.0.0

But apparently the default config for apps is for dependabot is to increase versions:

```

Why did it increase the version in the first place? If core is a module, it should just expand -- not increase from what I read.

---

> I think the issue was that Dependabot ran and then it got into a canary release

^^ That can't be the case. Canary releases _only_ happen on PR Merge or Push to Main branch. Or am I not understanding correctly?

> If this fixes that via process, then ok.

^^ I think it will

> "dotenv-webpack": "^2.0.0", ... which should keep the version between under 3.

^^ Hmm, maybe I'm not following here. You are correct, that `^` will not allow Yarn to install or reference a major version > 2. 

**However,** Dependabot is _always_ making recommends about _the most recent releases._ It's trying to keep us updated and has some (meh) algorithm for determining the compatibility of any new release. It's **not** following semver. **Dependabot is** trying to upgrade everything all the time.

Is that more helpful?